iocore/net/I_NetVConnection.h

```diff
@@ -566,6 +573,21 @@ class NetVConnection : public VConnection
     is_transparent = state;
   }
 
+  /// Set the context and return context value
+  NetVConnectionContext_t
+  set_context(NetVConnectionContext_t context)
```

Comment:
The return value of this is never used, so can we remove it? If the expectation is that the context is only set once, can we either
    - do it in the constructor and remove this
    - assert that it is unset here.